app/controllers/people_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @task = Task.all
+
+    Task.where(:completed_at => nil).count
+
+    render :index
+
+  end
+
+  def show
+    @task = Task.find(params[:id])
+
+    @person_id = @task.person_id
+    @person = Person.find(params[:id])
+
+    render :show
+  end
+
+  def tasks
+    @person = Person.find(params[:id])
+    @tasks = Task.where(:person_id => @person.id)
```

Comment:
Since you set up the Person model with a `has_many` for tasks, you can utilize the `@person` instance to retrieve any associated tasks: `@person.tasks` rather than retrieving them from the Task model.